linode/manager
# fix: [M3-7543] - AGLB Rule Drawer - Session Stickiness TTL - Use Seconds instead of Milliseconds

## Description 📝
Makes some fixes and refinements in the **AGLB Rule Drawer**

## Changes  🔄
- If you edit a rule, the `Save` button will be disabled until a change is made ⛔️
- Makes the Session Stickiness value use **seconds** instead of **milliseconds** (I incorrectly assumed this was milliseconds when I built this)⏲️

## Preview 📷

> [!important]
> No UI changed other than the fact the button is now disabled if no changes are made (when editing)

![Screenshot 2023-12-06 at 12 14 45 PM](https://github.com/linode/manager/assets/115251059/b45707f3-a5d1-4645-bc7d-85efbfe30a79)

## How to test 🧪

### Prerequisites
- Use the `dev-test-aglb` account to turn on the MSW

### Verification steps 

> [!note]
> The API seems to be offline right now, but you can still review this PR's changes

- Go to http://localhost:3000/loadbalancers/6/routes
- Verify you an _add_ a route and customize the Session Stickiness TTL
  - Inspect the payload and verify the TTL sent is in **seconds** and is the correct value
- Verify you an _edit_ a route and customize the Session Stickiness TTL
  - Inspect the payload and verify the TTL sent is in **seconds** and is the correct value

## As an Author I have considered 🤔
